maps/skirmishes/endless_arena_8p.xml

```diff
@@ -42,7 +42,9 @@
   "AllyView": false,
   "CircularMap": true,
   "Description": "Imagine a perfectly symmetrical world... \nWhere harmony is the most important virtue...\n\nCan you survive this seemingly endless war?\nCan you dominate the world?\n\n[color=\"yellow\"]Welcome to the Endless Arena![/color]\n\nFeatures:\n  * Automatic adjustment of allies/enemies depending on their current military power\n  * Fair support for weaker players\n  * Unique, strategy oriented gameplay\n  * High difficulty due to equilibrium oriented design\n  * The key to victory is to keep equlibrium and use your cunning. You'll have to use tricks and art of wars rather than sheer military power.\n\nNotes: recommended 64bit version of 0 A.D. Alternatively you can lower population limit in game setup if out of memory occurs during large battles.\n\nVersion  : 1.5.5\nMap Created  : 10/27/2024\nLast Edited  : 1/14/2025\nAuthor  : Przemek Błaszczyk",
-  "Keywords": [],
+  "Keywords": [
```

Comment:
@culturedcait I added the keyword so now it will show up when the filter is used in the game settings menu.